cbits/cbits.c

```diff
@@ -0,0 +1,56 @@
+#include <stdint.h>
+#include <string.h>
+#include <limits.h>
+#include <assert.h>
+
+/* Taken from:
+ *
+ * <http://stackoverflow.com/questions/776508/best-practices-for-circular-shift-rotate-operations-in-c>
+ *
+ * Intel is little-endian, we have to rotate right
```

Comment:
Yeah, it's fine since mask is now stored passed a `Word32`.  Anyway I'll clear up the docs a bit.  Thanks again for the PR!